client/state/themes/actions/activate-theme.js

```diff
@@ -56,7 +58,20 @@ export function activateTheme(
 				if ( error.error === 'theme_not_found' ) {
 					dispatch( errorNotice( translate( 'Theme not yet available for this site' ) ) );
 				} else {
-					dispatch( errorNotice( translate( 'Unable to activate theme. Contact support.' ) ) );
+					dispatch(
+						errorNotice(
+							translate(
+								'Unable to activate theme. {{contactSupportLink}}Contact support{{/contactSupportLink}}.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'To change your VAT number, {{contactSupportLink}}please contact support{{/contactSupportLink}}.' )` *ES Score: 6*